ext/templates-modals.html

```diff
@@ -1310,9 +1310,17 @@ <h1 class="modal-title">Pronunciation Dictionaries</h1>
             </div>
             <textarea autocomplete="off" spellcheck="false" id="generate-anki-notes-textarea" class="no-wrap margin-above" data-tab-action="indent,4"></textarea>
             <div class="generate-anki-notes-test-container margin-above">
-                <div>Active Anki Flashcard Format: <select id="generate-anki-flashcard-format"></select></div>
-                <div>Active Anki deck: <code id="generate-anki-notes-active-deck"></code></div>
-                <div>Active Anki model: <code id="generate-anki-notes-active-model"></code></div>
+                <div id="generate-anki-notes-buttons-wrapper">
+                    <div>
+                        <div>Active Anki Flashcard Format: <select id="generate-anki-flashcard-format"></select></div>
+                        <div>Active Anki deck: <code id="generate-anki-notes-active-deck"></code></div>
+                        <div>Active Anki model: <code id="generate-anki-notes-active-model"></code></div>
+                    </div>
+                    <div>
+                        <button type="button" class="low-emphasis" id="generate-anki-notes-parse-button">Parse Words</button>
```

Comment:
What does this do? Can you add a `title` field?

Reply:
Added.